lib/runner/extensions/request.command.js

```diff
@@ -13,17 +13,20 @@ module.exports = {
 
                 // helper function to trigger `response` callback anc complete the command
                 complete = function (err, nextPayload) {
-                    // trigger `response` callback
-                    // nextPayload.response will be empty for error flows
-                    this.triggers.response(
-                        err,
-                        nextPayload.coords,
-                        nextPayload.response,
-                        nextPayload.request,
-                        // this `item` is resolved and mutated here
-                        nextPayload.item,
-                        nextPayload.cookies
-                    );
+                    // nextPayload will be empty for unhandled errors
+                    if (nextPayload) {
+                        // trigger `response` callback
+                        // nextPayload.response will be empty for error flows
+                        this.triggers.response(
```

Comment:
The `.response` call here could take `nextPayload` directly. This looks verbose.

Reply:
`triggers.response` is the public `response` callback. This the signature of it.

Reply:
Fine for now, but this could be looked into later.